carbon/org.wso2.developerstudio.eclipse.artifact.carbon.ui.bundle/src/org/wso2/developerstudio/eclipse/artifact/carbon/ui/bundle/ui/wizard/CarbonUIbundleWizard.java

```diff
@@ -61,19 +61,20 @@
 
 public class CarbonUIbundleWizard extends AbstractWSO2ProjectCreationWizard {
 	private static final String PROJECT_WIZARD_WINDOW_TITLE = "New Carbon UI Bundle";
-    private static final String CARBON_UI_BUNDLE_PROJECT_NATURE = "org.wso2.developerstudio.eclipse.artifact.carbon.ui.bundle.project.nature";
+	private static final String CARBON_UI_BUNDLE_PROJECT_NATURE = "org.wso2.developerstudio.eclipse.artifact.carbon.ui.bundle.project.nature";
 	private static CarbonUiModel uibundleModel;
 	private  IProject project;
 	private Map<String, Text> controlersMap ;
 	private static IDeveloperStudioLog log=Logger.getLog(Activator.PLUGIN_ID);
 	private String projectName="";
 	
-    public CarbonUIbundleWizard(){
-     setUibundleModel(new CarbonUiModel());
-     setModel(uibundleModel);
-     setWindowTitle(PROJECT_WIZARD_WINDOW_TITLE);
-     setDefaultPageImageDescriptor(CarbonUIImageUtils.getInstance().getImageDescriptor("carbon-ui-bundle-wizard.png"));
-    }
+	public CarbonUIbundleWizard() {
```

Comment:
These are formatted using WSO2 formatter and I have checked them. What is the exact issue need to be fixed here? 
Thanks